libs/sdk-ui/src/base/localization/TranslationsCustomizationProvider/utils.ts

```diff
@@ -99,3 +99,26 @@ export const removeAllInsightToReportTranslations = (
         ...translations,
     };
 };
+
+/**
+ * The function to remove all translation keys contains suffix is "|report", "|insight", "._measure" or "._metric"
```

Comment:
Should we rewrite as follows to be somewhat clearer and more grammatically correct?

`The function to remove all translation keys contains suffix is "|report", "|insight", "._measure" or "._metric"`

↓

`The function to remove all translation keys that contain special suffixes`